src/migration/MigrationExecutor.ts

```diff
@@ -388,9 +388,11 @@ export class MigrationExecutor {
     }
 
     /**
-     * Reverts last migration that were run.
+     * If until is undefined, it reverts only the last executed migration.
+     * If until is "0", it reverts all migrations.
+     * Otherwise, it reverts all migrations executed after the migration with the specified name.
      */
-    async undoLastMigration(): Promise<void> {
+    async revertMigration(until?: string): Promise<void> {
```

Comment:
Personally I think object arguments are more preferable as it allows extending the parameters without breaking usages all over the codebase. Wdyt?
This is not crucial or a blocker btw, feel free to ignore